kong/conf_loader.lua

```diff
@@ -260,7 +260,7 @@ local _nop_tostring_mt = {
 
 -- Validate properties (type/enum/custom) and infer their type.
 -- @param[type=table] conf The configuration table to treat.
-local function check_and_infer(conf)
+local function check_and_infer(conf, opts)
```

Comment:
This would have been more correct:

```lua
if not opts.starting then
  log.disable()
end

-- ...

-- end of the function
```lua
if not opts.starting then
  log.enable() -- re-enable
end
```
```